src/components/content-box/_content-box.scss

```diff
@@ -34,7 +34,7 @@ $include-html: false !default;
 
     &--spaced {
       .mint-content-box__content, .mint-content-box__footer {
-        @media (min-width: $breakpointMedium) {
+        @include mintBreakpoint(medium-up) {
```

Comment:
This is not variable this is breakpoint name and to follow naming convention we have I think its better to keep it like this.
eg: `.mint-hide-for-small-only`
WDYT ?